### DISCUSSION TITLE
The ability to see how much folders weigh without going to the "Properties"

### DISCUSSION THREAD

**Discussion by modin523** [valinet/ExplorerPatcher#3419] [Ideas]
![Screenshot 2024-06-24 184547](https://github.com/valinet/ExplorerPatcher/assets/173688451/6d571e24-397e-4024-a5f1-25b94853dca7)

**Comment by pyrates999** [comment]
That would require Microsoft to add such a feature: https://answers.microsoft.com/en-us/windows/forum/all/not-able-to-see-folder-size-in-size-column/2cfbde76-3e20-4e80-bd8d-e89f3a863988
